src/schema/firefox-schemas-import.js

```diff
@@ -1,4 +1,5 @@
 import fs from 'fs';
+import { join as joinPath } from 'path';
```

Comment:
Just that `path` was already being used. I can do that and rename those variables.